model/openapi.yml

```
@@ -996,6 +996,25 @@ components:
                 statusCode: 401
                 error: Unauthorized
                 message: ApiKey not enabled
+    forbidden:
+      description: 403 response when the API key does not match the agent
```

For cases where the API key is invalid we return 401, right?
```suggestion
      description: 403 response when the API key is valid but does not match the agent
```